docs/ops/infrastructure/Loop_5.md

```diff
@@ -144,16 +144,14 @@ Loop operation description in the IR also has several special sections: `body`,
 
 **Body Inputs**
 
-* **Current iteration**: A scalar tensor of `int64` type specifying the current iteration number. *Required*.
-
-* **Multiple other inputs**: tensors of different types and shapes. *Optional*.
+* **Multiple inputs**: tensors of different types and shapes. One of the inputs corresponds to current iteration number input (this input is optional). This input is marked in the port_map with attribute `purpose = "iteration_number"`. *Optional*.
 
 
 **Body Outputs**
 
 * **Condition**: A scalar tensor of `boolean` type specifying whether to execute the next iteration or not. `True` means to continue execution.
```

Comment:
**Condition** -> **ExecutionCondition** or **BodyExecutionCondition**

Reply:
We should delete this required output due to `purpose = "execution_condition" attribute

Reply:
Done